src/stars.css

```diff
@@ -0,0 +1,17 @@
+.page__body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
```suggestion
}

.stars--1 :nth-child(-n + 1),
... ,
.stars--5 :nth-child(-n + 5) {
  background-image: url(./images/star-active.svg);
}
```

Reply:
Consider using commas to group selectors

Reply:
as i understood i have to create only 2 files, one for general styles and another for active stars and just add all the selectors modification separated with coma? then import these 2 files in main file style.css